airbyte-integrations/connectors/destination-mysql/Dockerfile

```diff
@@ -10,11 +10,34 @@ RUN tar xf ${APPLICATION}.tar --strip-components=1 && rm -rf ${APPLICATION}.tar
 
 FROM airbyte/integration-base-java:dev
 
+RUN yum install -y python3 python3-devel jq sshpass git && \
+  alternatives --install /usr/bin/python python /usr/bin/python3 60 && \
+  python -m ensurepip --upgrade && \
+  pip3 install dbt-mysql==1.0.0
+
+# Luckily, none of normalization's files conflict with destination-mysql's files :)
+# We don't enforce that in any way, but hopefully we're only living in this state for a short time.
+COPY --from=airbyte/normalization:dev /airbyte /airbyte
+# Install python dependencies
+WORKDIR /airbyte/base_python_structs
+RUN pip3 install .
+WORKDIR /airbyte/normalization_code
+RUN pip3 install .
+WORKDIR /airbyte/normalization_code/dbt-template/
+# Download external dbt dependencies
+# amazon linux 2 isn't compatible with urllib3 2.x, so force 1.26.15
+RUN pip3 install "urllib3<2"
+RUN dbt deps
+
 WORKDIR /airbyte
 
 ENV APPLICATION destination-mysql
+ENV AIRBYTE_NORMALIZATION_INTEGRATION mysql
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.1.20
+LABEL io.airbyte.version=0.1.22
```

Comment:
Looks like we only published v0.1.21 for destination-mysql-strict-encrypt... that was probably an oversight?